src/content/self-hosted/install/backup-restore.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Backup and Restore
+description: Backup and Restore your Okteto instance
+id: backup
+sidebar_label: Backup and Restore
+---
+
+Backup and restore planning is essential to ensure the availability and integrity of your Okteto instance in case of unexpected failures or disaster recovery.
+This guide outlines the steps to recover an Okteto instance in such scenarios.
+
+## Critical Resources
+
+To effectively recover an Okteto instance, focus on the following critical resources:
+
+- **Service Accounts**: Preserve service accounts associated with users. These service accounts reside within the `okteto` namespace and are labeled with `dev.okteto.com: "true"`.
+- **Secrets**: Recreate all secrets within the `okteto` namespace.
+- **Custom Resources**: Restore Custom Resources (CRs) corresponding to Okteto's CRDs. They can be identified with the label `app.kubernetes.io/part-of: okteto`.
+
+## Recovery Process
+
+You can follow two different recovery processes:
+
+- **Light recovery processes**: recovers all the users’ metadata, tokens and secrets, but it doesn’t recover users’ namespaces and volumes. All this information is available on the `okteto` namespace.
+- **Full recovery processes**: recovers not only the `okteto` namespace, but also all the users’ namespaces and preview environments, including data in persistent volumes.
+
+### Light Recovery Process
+
+If you do not wish to retain users’ namespaces and their application data, migrating the resources in the `okteto` namespace is sufficient. This is the fastest option, but your users must recreate their development environments.
+
+Follow these steps:
+
+- **Backup the okteto namespace**: backup the [critical resources](self-hosted/administration/configuration.mdx#subdomain) in your `okteto` namespace.
+- **Restore Service Accounts, Secrets, and Custom Resources**: restore service accounts, secrets, and Okteto custom resources in the `okteto` namespace.
+- **Reinstall Okteto**: reinstall the Okteto Helm chart with the same configuration as your previous Okteto instance.
```

Comment:
I'm confused here. The first step in the light _recovery_ process is to _backup_ the okteto namespace. Shouldn't that happen before a recovery process? I would expect the first step to be _restore_ the okteto namespace that was previously backed up.

Then I'm also not clear on how the restoration works. If there's an unexpected failure of some kind, where are these resources being restored that we can then reinstall Okteto via the helm chart? Are these being restored directly into some data store or something?

Reply:
It's restored in the previous step:
```
**Restore Service Accounts, Secrets, and Custom Resources**: restore service accounts, secrets, and Okteto custom resources in the `okteto` namespace.
```